health-sdk/sdk/src/main/java/net/gini/android/health/sdk/paymentcomponent/PaymentComponent.kt

```diff
@@ -23,7 +23,7 @@ import net.gini.android.health.sdk.paymentprovider.getPaymentProviderApps
  *
  * It requires a [GiniHealth] instance and a [Context] (application or activity) to be created.
  */
-class PaymentComponent(private val context: Context, private val giniHealth: GiniHealth) {
+class PaymentComponent(private val context: Context, internal val giniHealth: GiniHealth, private var configuration: PaymentComponentConfiguration = PaymentComponentConfiguration()) {
```

Comment:
@llevente 
Is it mistake, or we changed something that we need to use internal against private?

Reply:
@GeraltRiv  we have to get the language from GiniHealth through the payment component so it has to be accessable. Initially i made a getter for the private field, but Alpar suggested it can be internal, so the module could see it.